fiware-region-sanity-tests/commons/results_analyzer.py

```diff
@@ -47,24 +56,22 @@ def get_results(self):
 
         testsuite = doc.getElementsByTagName("testsuite")[0]
 
-        # Print the total tests and failures
-        print "{} [Tests: {}, Errors: {}, Failures: {}, Skipped: {}]".format(
-            testsuite.getAttribute("name"), testsuite.getAttribute("tests"), testsuite.getAttribute("errors"),
-            testsuite.getAttribute("failures"), testsuite.getAttribute("skip"))
-
-        # Total Tests
-        self.test += int(testsuite.getAttribute("tests"))
-
-        # Total Fail and Error Tests
-        self.failures += int(testsuite.getAttribute("failures")) + int(testsuite.getAttribute("errors"))
+        # Print a summary of the test results
```

Comment:
Currently, get("name") gives "nosetests", so it is completely useless in this summary.